addons/link_tracker/views/link_tracker_views.xml

```diff
@@ -41,10 +45,11 @@
                         </div>
                         <div class="oe_title mw-100">
                             <label for="code" string="Link Tracker"/>
-                            <h2 class="d-flex flex-wrap fw-normal">
+                            <h2 invisible="not id" class="d-flex flex-wrap fw-normal">
                                 <field name="short_url_host" nolabel="1" readonly="1" class="text-break w-auto me-1"/>
-                                <field name="code" nolabel="1" readonly="not code" invisible="not code" class="o_input w-100 w-sm-50 w-lg-25"/>
+                                <field name="code" nolabel="1" readonly="not id" invisible="not id" required="id" class="o_input w-100 w-sm-50 w-lg-25"/>
```

Comment:
`required="id"` is a little strange, I guess it doesn't work anymore if we manually remove the code after the link is created?

Should be motivated in the commit message if possible.